charts/templates/configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "link-shortener.fullname" . }}-configmap
+  namespace: link-shortener{{ .Values.django.env.run_env }}
```

Comment:
Seems like the namespace scheme for the systems team follows `applicationname${ENV}` i.e. link-shortenerprod, link-shortenertest?

Just want to make sure this isn't a typo

Reply:
@cachemeoutside That's the format we've been using.  For example, https://github.com/UCLALibrary/gitops_kubernetes/blob/main/apps/systems-team-prod-environment-values.yaml#L26 references `cataloging-statisticsprod` and https://github.com/UCLALibrary/gitops_kubernetes/blob/main/apps/systems-team-prod-environment-values.yaml#L59 uses `lbsprod`.